dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -533,6 +536,9 @@ def runCrossValidation(self, jobId, interfaces):
         if interfaces.jobDb.getSubmissionById(submissionId).number_of_errors == 0:
             interfaces.jobDb.setPublishableFlag(submissionId, True)
 
+        # Mark validation complete
+        interfaces.errorDb.markFileComplete(jobId, "")
```

Comment:
Should the `markFileComplete` method be updated to use a default for file name in the same way that `createFileIfNeeded` does? (see the question on that one re: `None` vs a blank string)?